### PR TITLE
fixed outdated flags

### PR DESCRIPTION
**Pull request by japandotorg**
### Description of the changes

- Fixed outdated flags in the worldmaps trivia list - Mauritania, Peru, Costa Rica, Bolivia
- Added the Palestine in the list (As of July 2019, 138 of the 193 United Nations (UN) members have recognized Palestine.)
- Fixed the name for Guinea Bissau from Guinea-Bissau to Guinea Bissau
- Added aliases to Republic of Congo (Roc) ,  Democratic Republic of the Congo (Drc) , Central African Republic (Car)

### Have the changes in this PR been tested?

Yes

### CODE REVIEW

File: redbot/cogs/trivia/data/lists/worldflags.yaml [hunk 1]
```diff
@@ -177,7 +178,7 @@ What country is represented by this flag? https://i.imgur.com/RTVaeeT.png:
 What country is represented by this flag? https://i.imgur.com/RgnmHkX.png:
 - Chile
 What country is represented by this flag? https://i.imgur.com/RhbzmE9.png:
-- Guinea-Bissau
+- Guinea Bissau
```
Comment (Kreusada) [inline redbot/cogs/trivia/data/lists/worldflags.yaml:183]:
```suggestion
- Guinea-Bissau
- Guinea Bissau
```

Guinea-Bissau is recognized with a hyphen as it always has been, the hyphonated version needs to stay. The non-hyphonated version may also be added.

File: redbot/cogs/trivia/data/lists/worldflags.yaml [hunk 2]
```diff
@@ -140,6 +140,7 @@ What country is represented by this flag? https://i.imgur.com/LgwBULL.png:
 What country is represented by this flag? https://i.imgur.com/Ls07jJq.png:
 - Democratic Republic of the Congo
 - Dem. Rep. of Congo
+- Drc
```
Comment (Kreusada) [inline redbot/cogs/trivia/data/lists/worldflags.yaml:143]: In order to maintain aliasing consistency within the various trivia lists, you should probably add an additional alias that separates this abbreviation with full stop characters ("DRC" + "D.R.C."). You should also apply this change to the other abbreviating aliases you have added in this PR.

Comment (Kreusada) [inline redbot/cogs/trivia/data/lists/worldflags.yaml]:
```suggestion
- DRC
- D.R.C.
```
In order to maintain aliasing consistency within the various trivia lists, you should probably add an additional alias that separates this abbreviation with full stop characters ("DRC" + "D.R.C."). You should also apply this change to the other abbreviating aliases you have added in this PR.

File: redbot/cogs/trivia/data/lists/worldflags.yaml [hunk 3]
```diff
@@ -225,10 +226,12 @@ What country is represented by this flag? https://i.imgur.com/YaA07yt.png:
 - Belize
 What country is represented by this flag? https://i.imgur.com/Yvry0ap.png:
 - Central African Republic
+- Car
```
Comment (Kreusada) [inline redbot/cogs/trivia/data/lists/worldflags.yaml]:
```suggestion
- CAR
- C.A.R.
```

File: redbot/cogs/trivia/data/lists/worldflags.yaml [hunk 4]
```diff
@@ -225,10 +226,12 @@ What country is represented by this flag? https://i.imgur.com/YaA07yt.png:
...
 What country is represented by this flag? https://i.imgur.com/ZZJK5He.png:
 - Republic of the Congo
 - Congo
 - Republic of Congo
+- Roc
```
Comment (Kreusada) [inline redbot/cogs/trivia/data/lists/worldflags.yaml]:
```suggestion
- ROC
- R.O.C.
```

**Review by Kreusada — COMMENTED** [review]
The flag changes look good. I just had a few things to add/adjust in some of the changes you've made.

**Comment by japandotorg** [comment]
I've made the changes as per suggested ^.^

**Review by Flame442 — APPROVED** [review]
Thanks for updating these!

**Pull request merged**